src/default-list.js

```diff
@@ -1,18 +1,27 @@
-export default function defaultList(item, i) {
+import { toggleComplete } from './complete.js';
+import { createMenu, delMenu } from './dropdown.js'; // eslint-disable-line
```

Comment:
Kindly remove all disabled Linters, Linting helps in detecting errors, writing clean and professional code ✊ 

Please check for other occurrences 🙂

Reply:
If I have two functions that intrinsically refer to each other, how do I get around it?